### ISSUE TITLE
Run without visual

### ISSUE THREAD

**Issue by caa318** [qiao/PathFinding.js#147]
Is it possible to run this without loading the visual?
I load in a large matrix of 1's and 0's and **all I want is the program to print the path coordinates**.  Right now, it takes a long time for the grid to generate and the path to do its visual even though in the stats it says it takes less than a second.
Thank you!

**Comment by brean** [comment]
Yes, it is. You can run the code without any visual output. Depending on your use case.
You can for example download and build the code with gulp and run it on your local machine. Take a look at the src/ folder.
Another option is to go to http://qiao.github.io/PathFinding.js/visual/ open the console (F12 in Chrome or Internet Explorer) and use the already compiled code - for example:
```
var grid = new PF.Grid(10, 10); 
grid.getNodeAt(2,0).walkable = false; // simple obstacle
var finder = new PF.AStarFinder();
var path = finder.findPath(0, 0, 9, 9, grid);
console.log(JSON.stringify(path));
```
